src/main/java/com/deavensoft/paketomat/user/UserController.java

```diff
@@ -91,5 +92,11 @@ public int deleteUser(@PathVariable(name = "id") Long id) throws NoSuchUserExcep
             return 1;
         }
     }
+    @GetMapping(path="/pay/{id}")
```

Comment:
Does this make sense to be in UserController? 
According to the implementation, the path would be api/users/pay/{id}. It's better to be in the PackageController as api/packages/pay?id={id}.

Reply:
My idea was that the user need to pay it, but it makes more sense the logic to be in PackageController

Reply:
As path I expect something like api/packages/**pay?id={id}**